.github/workflows/tests_and_qc_frontend.yml

```diff
@@ -0,0 +1,38 @@
+# TODO: get to run in GH action (right now times out), then reactivate "on"
+# TODO: (a) start backend within GH action, or (b) use dev's backend URL (won't always work)
+# TODO: expose params to workflow_dispatch
+name: Tests and QC - Frontend
+on:
```

Comment:
## GH action tests status
@Sigfried I got this simple [GH action working](https://github.com/jhu-bids/TermHub/actions/runs/6092164533/job/16529852770). There's still work to do but this is a good start, maybe good to merge now and I can continue working on these actions.

GH action test setups
- [x] 1. frontend, dev/prod, manually
- [x] 2. frontend, dev/prod, daily
- [ ] 3. frontend/backend, local, on PR & pushes to develop
- [ ] 4. frontend, dev/prod, on deploy
- [ ] 5. backend, dev/prod, on deploy

Some additional things that might be nice
- [ ] i. Allow automated rollbacks (just prod? so that we can troubleshoot on dev)
- [ ] ii. Allow params for GH action (e.g. running manually, can select to run on just dev or prod, etc)

I'll need to update a GH issue w/ this. The next most important one is (4). The other ones can wait a bit more. I might be able to do the backend part of (3) and (5) very soon.